edx_zoom/zoom_consumer.py

```diff
@@ -84,21 +85,24 @@ def _get_context_for_template(self):
         ctx = super(ZoomXBlock, self)._get_context_for_template()
         ctx['missing_credentials'] = self.lti_provider_key_secret[0] is None
         ctx['ask_to_send_username'] = ctx['ask_to_send_email'] = True
-        try:
-            ctx['is_studio'] = self.runtime.service(self, 'completion') is None
-        except Exception:
-            ctx['is_studio'] = True
         return ctx
 
     def student_view(self, context):
         fragment = Fragment()
         lti_loader = ResourceLoader('lti_consumer')
         loader = ResourceLoader(__name__)
-        context.update(self._get_context_for_template())
+        context = self._get_context_for_template()
```

Comment:
Is this change necessary? The context passed in will be lost. Is that intended?

Reply:
It would work fine in LMS without this change, but if someone is calling xblock View directly and passes some query params then context becomes QueryDict object and the `update` method adds zoom xblock context to a query dict which the template is unable to read because it renders as: `{'key': ['value']}`. Since we are not using any thing from the context passed so i simply skipped it.